src/services/TreeInfo.service.js

```diff
@@ -0,0 +1,58 @@
+import Vue from 'vue'
+import wiki from 'wikijs'
+
+export const treeInfoService = {
+    getTreeInfo,
+    getImageLink,
+    getPageLink
+}
+
+const treedata = require('../assets/trees/treenames.json')
+
+/**
+ * lookup this tree in the json file and get other details
+ * 
+ * see end of file to view the fields
+ * 
+ * @param {*} latin_code the latin code to lookup in the json
```

Comment:
NIT: `latin-code` type is a string, right?

Reply:
Right